test/SourceKit/CompileNotifications/diagnostics.swift

```diff
@@ -87,3 +87,42 @@
 // RUN: %sourcekitd-test -req=track-compiles == -req=syntax-map %s | %FileCheck %s -check-prefix=SYNTACTIC
 // SYNTACTIC: key.syntaxmap:
 // SYNTACTIC_NOT: key.notification: source.notification.compile-did-finish
+
+// RUN: %sourcekitd-test -req=track-compiles == -req=sema %S/Inputs/descriptive-error.swift -- -Xfrontend -enable-descriptive-diagnostics -Xfrontend -diagnostic-documentation-path -Xfrontend /educational/notes/path/prefix %S/Inputs/descriptive-error.swift | %FileCheck %s -check-prefix=DESCRIPTIVE
+// DESCRIPTIVE: key.notification: source.notification.compile-did-finish
+// DESCRIPTIVE-NEXT: key.diagnostics: [
+// DESCRIPTIVE-NEXT:   {
```

Comment:
This test is being pretty strict about what it expects, which makes it potentially brittle about changes.  I would suggest simplifying this to something like this

```
// DESCRIPTIVE:          key.description: "non-nominal type
// DESCRIPTIVE:          key.educational_notes: [
// DESCRIPTIVE-NEXT:       "/educational/notes/path/prefix/nominal-types.md"
// DESCRIPTIVE-NEXT:     ]
```

Reply:
👍